src/docs/maturity/use-cases.md

```diff
@@ -58,17 +58,19 @@ We also overlay commit and deployment behavior to correlate issues with applicat
 
 For service logs, we store logs for up to 90 days for Pro plan workspaces.
 
-Included in our planned near-term work for logging and observability on Railway are structured logs, OpenTelemetry compatible endpoints, and structured logs. 
+Included in our planned near-term work for logging and observability on Railway are improvements to structured logging compatibly, and OpenTelemetry compatible endpoints.
 
-It is common for teams who wish to have additional observability to use an additional monitoring tool that maintains a longer time horizon of data such as New Relic, Sentry, or Datadog. Within projects, deploying a Datadog Agent is as easy as using a provided Dockerfile and providing your Datadog API Keys.
+It is common for teams who wish to have additional observability to use an additional monitoring tool that maintains a longer time horizon of data such as New Relic, Sentry, or Datadog. Within projects, deploying a Datadog Agent is as easy as deploying the template and providing your Datadog API Keys.
 
 ### Networking
 
-Railway doesn't have a hard bandwidth limit within Projects and the broader internet. 
+Railway doesn't have a hard bandwidth limit to the broader internet. 
 
-We throttle outbound bandwidth when you exceed 100GB/month, but we happily lift that limit for legitimate customers. As such, we have had projects handle unexpected traffic and features on major media publications. It is something we are very proud of. With that said, if you need to control where your traffic is allowed to come from such as setting up firewall rules, we recommend setting up Cloudflare to handle this aspect of your application.
+We will get in contact when your outbound bandwidth exceed 100GB/month, but we happily lift that limit for legitimate customers. As such, we have had projects handle unexpected traffic and features on major media publications. It is something we are very proud of. With that said, if you need to control where your traffic is allowed to come from such as setting up firewall rules, we recommend setting up Cloudflare to handle this aspect of your application.
```

Comment:
Mostly unrelated to your changes here but I'd rework this entirely into:

```suggestion
We may throttle your outbound bandwidth and reach out to you when it exceeds 100GB/month to ensure the legitimacy of your workloads. If you need to control where your traffic is allowed to come from such as setting up firewall rules, we recommend setting up Cloudflare or an external load balancer/L7 application firewall to handle it.
```

"As such, we have had projects handle unexpected traffic and features on major media publications. It is something we are very proud of." is not something we should have in docs from a tone perspective, specifically the latter bit